src/v3/preact.config.js

```diff
@@ -10,54 +10,47 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* eslint-disable import/no-extraneous-dependencies */
-import buffer from 'buffer';
+/* eslint-disable import/no-extraneous-dependencies,no-param-reassign */
```

Comment:
Could we still keep the eslint-disable for `no-param-reassign` at the places they occur?

Reply:
Actually this file's primary purpose is to modify the `config` object so it's throughout